mitodl/micromasters
# Process proctored exam grade report from edX

#### Pre-Flight checklist


#### What are the relevant tickets?
Fix #4668

#### What's this PR do?
Add a management command for processing  proctored exam grade report form edX

#### How should this be manually tested?
In django shell set up an ExamRun that is currently schedulable. Make sure your user paid for the course.
Run the command `./manage.py import_edx_exam_grades 14.100`
